src/main/kotlin/io/slama/events/autoroles.kt

```diff
@@ -21,6 +21,7 @@ private val logger: Logger = LoggerFactory.getLogger("AutoRolesManager")
 class AutoRole(
     private val config: AutoRoleDTO,
     private val name: String,
+    private val guild: Guild,

```

Comment:
Should be changed to guildId because of JDA cache system deleting objects randomly.